make/test.mk

```diff
@@ -139,7 +139,7 @@ test-e2e-ci: get-test-namespace ./vendor
 	$(Q)sed -e 's|REPLACE_NAMESPACE|$(TEST_NAMESPACE)|g' ./deploy/test/role_binding_test.yaml | oc apply -f -
 	$(Q)sed -e 's|REPLACE_IMAGE|registry.svc.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:tektoncd-pipeline-operator|g' ./deploy/test/operator_test.yaml  | oc apply -f - --namespace $(TEST_NAMESPACE)
 	$(eval DEPLOYED_NAMESPACE := $(TEST_NAMESPACE))
-	$(Q)operator-sdk test local ./test/e2e --namespace $(TEST_NAMESPACE) --no-setup --go-test-flags "-v -timeout=15m"
+	$(Q)operator-sdk test local ./test/e2e --operator-namespace $(TEST_NAMESPACE) --no-setup --go-test-flags "-v -timeout=15m"
```

Comment:
could you add `--watch-namespace ""` as well
we need both.

from v17.2 we have 2 flags instead of the old `--namespace` as you noticed.

if we don't set `--watch-namespace=""` the operator under test will only watch one namesapce, that will be = value of `--operator-namespace`

Reply:
I think `--watch-namespace` is required if there is a `--up-local` flag as per this PR https://github.com/operator-framework/operator-sdk/pull/2617 information